.eslintrc.json

```diff
@@ -12,7 +12,7 @@
     "ecmaVersion": 2018,
```

Comment:
- [ ] I see that you have implemented the functionality of the calculator already. Kindly remove this code and keep it safe because you will need it in the next project milestone.